docs/DeveloperGuide.md

```diff
@@ -420,38 +481,75 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The provided student's name is invalid/cannot be found.
```

Comment:
The provided student’s name is invalid or cannot be found.

Reply:
Should be The inputted student's name is invalid/cannot be found for consistency right?